saucedemo_framework/tests/test_cart_page.py

```diff
@@ -0,0 +1,37 @@
+import pytest
+
+
+@pytest.mark.regression
+def test_continue_shopping(open_cart_page):
+    cart_page = open_cart_page
+    store_page = cart_page.continue_shopping()
+    assert store_page.is_secondary_header_located() is True, 'Wasn`t able to continue shopping'
+
+
+@pytest.mark.regression
+def test_remove_item_from_cart(open_cart_page_with_item):
+    cart_page = open_cart_page_with_item
+    cart_page.press_remove_button()
+    assert cart_page.is_remove_button_displayed() is False, 'Item was not removed from the cart'
+
+
+@pytest.mark.regression
+def test_burger_menu_cart(open_cart_page):
+    cart_page = open_cart_page
+    cart_page.open_burger_menu()
+    assert cart_page.is_burger_menu_located() is True, 'Burger menu was not opened'
```

Comment:
it is better to verify if burger menu has correct items in it

Reply:
Do I check all four in one test case? Or it can be done with page object method?

Reply:
Done